wfe/web-front-end.go

```diff
@@ -208,11 +211,15 @@ func (wfe *WebFrontEndImpl) sendError(response http.ResponseWriter, message stri
 	case http.StatusInternalServerError:
 		problem.Type = ServerInternalProblem
 	}
+
 	problemDoc, err := json.Marshal(problem)
 	if err != nil {
 		problemDoc = []byte("{\"detail\": \"Problem marshalling error message.\"}")
 	}
-	wfe.log.Debug("Sending error to client: " + string(problemDoc))
+
+	// AUDIT[ Improper Messages ] 0786b6f2-91ca-4f48-9883-842a19084c64
+	wfe.log.Audit(fmt.Sprintf("Improper HTTP request - %d - %s - %s", code, details, debug))
```

Comment:
This audit category should probably depend on the problem type. ServerInternalProblem -> Error Conditions. MalformedProblem and UnauthorizedProblem -> Improper Messages.